client/src/cli_args.rs

```diff
@@ -202,6 +208,28 @@ impl<'a, 'b> EncointerArgs<'b> for App<'a, 'b> {
 				.help("cid of the community currency in which tx fees should be paid"),
 		)
 	}
+	fn meetup_index_arg(self) -> Self {
+		self.arg(
+			Arg::with_name(MEETUP_INDEX_ARG)
+				.long("meetup-index")
+				.takes_value(true)
+				.required(false)
+				.value_name("MEETUP_INDEX")
+				.help("the meetup index for which to claim rewards"),
+		)
+	}
+
+	fn all_upto_arg(self) -> Self {
+		self.arg(
+			Arg::with_name(MEETUP_INDEX_ARG)
```

Comment:
copy-paste error. guess you didn't test this? 🤭

Reply:
oh good catch. i actually did test an it worked, but just with one meetup, maybe this is why is still worked. or could it maybe have worked because the `.long` is set correclty?